### PR TITLE
Sram enable for AMD platforms

### PR DESCRIPTION
**Pull request by DINESHKUMARK1**

### CODE REVIEW

**Review by sunilkumardommati — APPROVED** [review]

File: src/platform/amd/rembrandt/rembrandt.x.in [hunk 1]
```diff
@@ -100,8 +100,11 @@ MEMORY
         org = SOF_STACK_END,
         len = SOF_STACK_BASE - SOF_STACK_END,
 sof_sdram1 :
+        org = DRAM1_BASE,
+        len = DRAM1_SIZE
+sof_sram1 :
         org = HEAP_BUFFER_BASE + HEAP_BUFFER_SIZE,
```
Comment (bhiregoudar) [inline src/platform/amd/rembrandt/rembrandt.x.in:106]: origin should be HEAP_SYSTEM_BASE

File: src/platform/amd/rembrandt/rembrandt.x.in [hunk 2]
```diff
@@ -100,8 +100,11 @@ MEMORY
...
-        len = DRAM1_SIZE- (HEAP_SYSTEM_SIZE +HEAP_SYS_RUNTIME_SIZE + HEAP_RUNTIME_SIZE + HEAP_BUFFER_SIZE + SOF_STACK_SIZE )
+        len = SRAM1_SIZE- (HEAP_SYSTEM_SIZE +HEAP_SYS_RUNTIME_SIZE + HEAP_RUNTIME_SIZE + HEAP_BUFFER_SIZE + SOF_STACK_SIZE )
```
Comment (bhiregoudar) [inline src/platform/amd/rembrandt/rembrandt.x.in:107]: Keep the length to SRAM1_SIZE

**Review by bhiregoudar — CHANGES_REQUESTED** [review]
Please modify as mentioned in the comments and resubmit.

**Comment by gkbldcig** [comment]
Can one of the admins verify this patch?

**Comment by lgirdwood** [comment]
test this please

**Comment by lgirdwood** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
I've run the CIs.

**Review by lgirdwood — COMMENTED** [review]
Commit messages are missing some detail regarding the changes but otherwise LGTM

**Comment by DINESHKUMARK1** [comment]
find the code changes and merge to SOF

**Review by bhiregoudar — APPROVED** [review]

**Review by sunilkumardommati — APPROVED** [review]

**Comment by lgirdwood** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
@wszypelt can you check CI, not expecting this PR to fail. Thanks !

**Comment by wszypelt** [comment]
@lgirdwood we have problem with internal service now ;/ 
It seems that we already know what is wrong, so I sincerely hope that everything will work today

**Review by iuliana-prodan — APPROVED** [review]

**Pull request merged**